packages/cli/tsconfig.json

```diff
@@ -9,8 +9,8 @@
     "typeRoots": ["./types", "./node_modules/@types"]
   },
   "extends": "../../tsconfig.base.json",
-  "include": ["./types", "src/**/*"],
-  "exclude": ["templates/*"],
+  "include": ["./types", "src/**/*", "test/**/*"],
+  "exclude": ["templates/*", "**/fixtures/**"],
```

Comment:
This is the one change that includes tests in `pnpm type-check`. Everything else is fixing type errors.